src/libertem/io/dataset/base/tiling_scheme.py

```diff
@@ -188,6 +188,8 @@ def validate(
     ):
         sig_shape = shape[1:]
 
+        return True
```

Comment:
Can we have a comment here that justifies disabling the validation code below?

Reply:
Hm, I'm wondering how that ended up here 😅 looks like a quick band-aid fix that should have been removed again. I'll try to keep the validation working, and otherwise put a proper justification.

Reply:
Nice catch! It turns out that the shape negotiation was in disarray, actually. It worked well because of a combination of bugs that led to an OK tile shape that got approved by the "True". I'm now working to find a good compromise using `get_max_io_size()`. Performance is not great yet for stddev, at least on my machine and Windows.

Reply:
Found a compromise that seems OK. The problem wasn't Windows, but a combination of not many cores, small L3 cache and less-than-ideal GPU support, I suspect way too small tiles.